modules/dnn/src/int8layers/convolution_layer.cpp

```diff
@@ -1384,10 +1386,36 @@ class ConvolutionLayerInt8Impl CV_FINAL : public BaseConvolutionLayerInt8Impl
         int nstripes = std::max(getNumThreads(), 1);
         Mat outputInt32 = Mat(shape(outputs[0]), CV_32S);
 
-        ParallelConv::run(inputs[0], outputInt32, weightsMat, outputMultiplier, biasvec, activationLUT, kernel_size, strides,
-                          pads_begin, pads_end, dilations, activ.get(), ngroups, nstripes, input_zp, output_zp);
+        // TODO! optimize the fastQConv by AVX/AVX2, need to rethink the Pack_layout.
+        // Currently, on x86 and Loongson platform, the fastQConv branch is slower than ParallelConv branch.
+        // So we keep it for now.
+#if CV_NEON && CV_NEON_AARCH64
+        if (!fastQConvImpl)
+        {
+            int K = outputs[0].size[1];
+            int C = inputs[0].size[1];
+
+            int conv_dim = CONV_2D;
+            if (inputs[0].dims == 3)
+                conv_dim = CONV_1D;
+            if (inputs[0].dims == 5)
+                conv_dim = CONV_3D;
+
+            CV_Assert(outputs[0].size[1] % ngroups == 0);
+            fastQConvImpl = initFastQConv(weightsMat, &biasvec[0], ngroups, K, C, kernel_size, strides, dilations,
+                                          pads_begin, pads_end, conv_dim, outputMultiplier, input_sc, input_zp,
+                                          output_sc, output_zp, per_channel);
+        }
 
-        outputInt32.convertTo(outputs[0], CV_8S);
+        runFastQConv(inputs[0], outputs[0], fastQConvImpl, nstripes, activ);
+#else
+        {
+            ParallelConv::run(inputs[0], outputInt32, weightsMat, outputMultiplier, biasvec, activationLUT, kernel_size, strides,
+                              pads_begin, pads_end, dilations, activ.get(), ngroups, nstripes, input_zp, output_zp);
+
+            outputInt32.convertTo(outputs[0], CV_8S);
+        }
+#endif
```

Comment:
Currently, on AVX/AVX2 platforms, the new branch(fastQConv) is slower than the old branch due to the lack of a better strategy. Therefore, the old branch(ParallelConv) is currently kept and I will continue to be optimized in 4.9 and remove the old branch.